### DISCUSSION TITLE
Flasher - will there be a possibility for individual builds?

### DISCUSSION THREAD

**Discussion by onki69** [EdgeTX/edgetx#925] [General]
Hi,

I really like EdgeTX with my Radiomaster TX 16s. In addition I use a Jumper T16 as trainer and in future also a Jumper T-Lite as trainer.

With the T16 I use Bluetooth for wireless trainer link. This is not yet possible yeat since I cannot manage to build an individual version (with noheli, Germen UI etc.).
It worked once but the firmware options were not what I needed but now all config info seems to be lost and i cannot find all commands to get my individual build done.

Will there be an option in the flasher to build such versions like it was in the OTX Companion. I don't like and need Companion and to me the individual builds are better located in the flasher.

Best regards
Onki

**Comment by raphaelcoeffic** [comment]
@onki69 yes, this is the plan. We are working the build platform itself right now, then on the integration with the flasher and/or companion.
